### PR TITLE
Fix failing Circle CI tests

### PR DESCRIPTION
**Pull request by jazzsequence**
Circle CI tests are failing at the PHP Unit step due to the lack of a recognized PUBKEY for the `apt` functions. (see https://app.circleci.com/pipelines/github/pantheon-systems/solr-power/1045/workflows/a5639025-0328-43b1-9af7-715a60903139/jobs/3320/parallel-runs/0/steps/0-105)

This PR adds a step in `config.yml` that adds the missing PUBKEY.

It also does some quality-of-life improvements using a newer Quay docker image and removing the _third_ call to run PHPUnit. (Only two are required.)

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -114,7 +118,6 @@ jobs:
           command: |
             bash bin/install-wp-tests.sh wordpress_test root '' 127.0.0.1 latest
             composer phpunit
-            WP_MULTISITE=1 composer phpunit
             rm -rf $WP_TESTS_DIR $WP_CORE_DIR
             bash bin/install-wp-tests.sh wordpress_test root '' 127.0.0.1 nightly true
-            composer phpunit
+            WP_MULTISITE=1 composer phpunit
```
Comment (CdrMarks) [inline .circleci/config.yml:123]: Is this change needed to fix the tests?

Comment (jazzsequence) [reply .circleci/config.yml:123]: I rearranged the unit tests. They were running three times:

1. Once through as a single site
2. Once through _on the same environment_ as a multisite
3. Once through _on a new environment_ as a single site

I think the third iteration was a mistake and that the multisite check should have run after the environment was wiped and rebuilt.

Comment (jazzsequence) [reply .circleci/config.yml:123]: The only change that was necessary to get the tests to pass was https://github.com/pantheon-systems/solr-power/pull/545/files#diff-78a8a19706dbd2a4425dd72bdab0502ed7a2cef16365ab7030a5a0588927bf47R105

This was just clean up and speeding up the PHP Unit step.

**Comment by jspellman814** [comment]
Can we sqaush commits before merging to `master`?

**Comment by jazzsequence** [comment]
@jspellman814 Squashing is allowed. I turned off auto-merge so I can explicitly squash.

**Review by CdrMarks — APPROVED** [review]
